packages/oui-select-picker/src/select-picker.html

```diff
@@ -1,8 +1,8 @@
 <input type="radio" class="oui-select-picker__input"
         name="{{:: $ctrl.name }}"
-        ng-attr-aria-labelledby="{{:: $ctrl.id }}-label"
-        ng-attr-aria-describedby="{{:: $ctrl.description && $ctrl.id + '-description' || undefined }}"
-        ng-attr-id="{{:: $ctrl.selectedValue ? $ctrl.id : undefined }}"
+        ng-attr-aria-labelledby="{{:: $ctrl.id }}Label"
+        ng-attr-aria-describedby="{{:: $ctrl.description && $ctrl.id + 'Description' || undefined }}"
```

Comment:
`ng-attr-aria-describedby="{{:: ($ctrl.description && $ctrl.id + 'Description') || undefined }}"` is more readable maybe :)